File: 5 - Loops/Solution.java
import java.util.Scanner;

public class Solution {
    public static void main(String[] args) {
        Scanner in = new Scanner(System.in);
        int N = in.nextInt();
        in.close();

        for (int i = 1; i < 11; i++) {
            System.out.println(N + " x " + i + " = " + N * i);
        }
    }
}

File: 5 - Loops/Solution.scala
object Solution {

    def main(args: Array[String]) {
        val sc = new java.util.Scanner (System.in);
        var N = sc.nextInt();
        
        (1 to 10).foreach((i: Int) => {
            println(N.toString + " x " + i.toString + " = " + (N * i).toString);
        })
    }
}
